packages/ActionBar/src/hooks/useSort/useSort.js

```diff
@@ -21,7 +21,7 @@ function sortData({ sortedFields, columns, data, locale }) {
   return sortedData;
 }
 
-export default function useSort({ data = null, columns }) {
+export default function useSort({ data = null, columns, maximumSort = null }) {
```

Comment:
Wondering what everyone thinks of the name `maximumSort`? Maybe `maximumSortFields` is better? `maximumSorts`? Or shorten it to `maxSorts`?